src/build/configure-guest.sh

```diff
@@ -77,6 +77,12 @@ prepare () {
       state: directory
       mode: 0755
     loop: "{{ ['dhcpd', 'traefik', 'authelia', 'adguardhome', 'netdata', 'heimdall', 'database', 'nextcloud', 'jellyfin'] | product(['config', 'data']) | list }}"
+  - name: ensure media directories exist
+    file:
+      path: "/pinanas/dist/{{ item[0] }}/{{ item[1] }}"
+      state: directory
+      mode: 0760
+    loop: "{{ ['media'] | product(['movies', 'tvshows']) | list }}"
```

Comment:
Duplicate with [src](https://github.com/yscialom/pinanas/tree/develop/src)/[tasks](https://github.com/yscialom/pinanas/tree/develop/src/tasks)/[jellyfin](https://github.com/yscialom/pinanas/tree/develop/src/tasks/jellyfin)/main.yaml

Reply:
I will edit this file.

Reply:
Could be simplified + mode 0770 would be preferred:
```yaml
  - name: ensure media directories exist
    file:
      path: "/pinanas/dist/media/{{ item }}"
      state: directory
      mode: 0770
    loop:
    - movies
    - tvshows
```

Reply:
FYI mode 0770 versus 0760 means the group (docker) will gain the eXecution rights. For directories, this right is needed to list (as in `ls`) its content.

Reply:
FYI2 mode traditionally is written with a leading zero to denote it is a number in base 8. This leading zero is sometimes used to carry a special right, called "the sticky bit".

To read the mode `0ugo`:
- `u`: user's permission (file owner)
- `g`: group's permission
- `o`: other's permission
where each digit is the sum of the following rights:
- `r`, 4: read
- `w`, 2: write
- `x`, 1: execute.

So `0760` means user can do all (4+2+1), group can read and write (4+2) but cannot list the content, others cannot do anything (0).